### PR TITLE
[Refactor] Remove unused imports

### PR DESCRIPTION
**Pull request by samar-hassan**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -28,7 +28,7 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.8"
 odin = "^2.9"
-django-oscar = {version = "^3.2.2", allow-prereleases = true}
+django-oscar = {version = "^3.2.3a1", allow-prereleases = true}
```
Comment (timsavage) [inline pyproject.toml:31]: I recommend using `"^3.2"`.

When you relock the poetry file this will resolve to the latest version that is `>=3.2.0 <4.0.0`

Comment (samar-hassan) [reply pyproject.toml:31]: This won't resolve to 3.2.3a1, which is necessary in this case

Comment (timsavage) [reply pyproject.toml:31]: Just checked and confirmed it does lock to 3.2.3a1 with:
`{version = "^3.2", allow-prereleases = true}`

Comment (samar-hassan) [reply pyproject.toml:31]: With `{version = "^3.2", allow-prereleases = true}`
I am getting version 3.2.2, not sure why that is.
image

**Pull request merged**